### PR TITLE
Add QuickJump Links

### PR DESCRIPTION
**Pull request by mcrmonkey**
Added quick jump links to some services so they are easier to find

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
 ## 2FA Audit
 Make sure 2FA or two step verification is enabled on all of the following accounts:
 
-- [ ] Google
+- [ ] Google - [Quick Jump](https://myaccount.google.com/security/signinoptions/two-step-verification)
```
Comment (alulsh) [inline README.md]: Hmm could we change to `[Google](https://myaccount.google.com/security/signinoptions/two-step-verification)` instead? I'm not a fan of the "quick jump" phrase. It reminds me of "click here" links - [here's a great overview of why you shouldn't use click here for links](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/).

File: README.md [hunk 2]
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
...
 - [ ] Amazon
-- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/)
-- [ ] GitHub
-- [ ] Dropbox
+- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/) - [Quick Jump](https://www.facebook.com/settings?tab=security&section=approvals)
```
Comment (alulsh) [inline README.md]: Change to `Facebook - [enable Login Approval](https://www.facebook.com/settings?tab=security&section=approvals)`

File: README.md [hunk 3]
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
...
+- [ ] GitHub - [Quick Jump](https://github.com/settings/security)
```
Comment (alulsh) [inline README.md]: Change to `[GitHub](https://github.com/settings/security)`

File: README.md [hunk 4]
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
...
+- [ ] Dropbox - [Quick Jump](https://www.dropbox.com/account/#security)
```
Comment (alulsh) [inline README.md]: Change to `[Dropbox](https://www.dropbox.com/account/#security)`

**Comment by alulsh** [comment]
Thanks for this PR and for wanting to improve this checklist, @mcrmonkey! Linking directly to the 2FA settings pages is a good improvement.

I left some line notes on this PR. Once those are addressed I'll gladly merge this PR.

**Comment by mcrmonkey** [comment]
Links have been updated as per recommendations
The [piece on click here links](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/) was a good read!

**Comment by alulsh** [comment]
Thanks again @mcrmonkey! I'm going to merge this now. This will definitely help people configure 2FA on their accounts.

**Pull request merged**